### PR TITLE
Percentage cut-off and file outdate time

### PR DESCRIPTION
**Pull request by reteppeter**
Made the overall percentage cut-off and file outdate time configurable, as well as implemented the percentage cut-off UI.

### CODE REVIEW

File: components/SummaryContainer.tsx [hunk 1]
```diff
@@ -32,15 +32,19 @@ export default function SummaryContainer(props: {
...
         <div>
             {props.loadingBackdrop}
         </div>
       <div className={styles.container}>
-      <div className={`${overallNan == false ? styles.summaryOverall : styles.summaryOverallGrey} ${styles.sharedCompProps} ${styles.summaryOverall}`}>
-            <h3 className={styles.overallTitleStyle}>Overall</h3>
-            <h2 className={styles.percentStyle} >{overallPercentStr}</h2>
+		<div className={`${styles.summaryComponent} ${styles.sharedCompProps}`}>
+			<h3 className={styles.summaryStyle}>Target ({config.targetPercentage}%)</h3>
+			<div className={`${overallStyle} ${overallColour} ${styles.smallSharedCompProps} ${styles.summaryOverall}`}>
```
Comment (alexdelicato) [inline components/SummaryContainer.tsx:43]: Looks like the headings and sub-content for the summary container page aren't aligned horizontally
image

**Review by alexdelicato — APPROVED** [review]
Design looks really good aside from some css alignment. Think some of the css changes in SummaryContainer.module.css will conflict with Leanne's current PR so best to get her to look at this as well.

**Review by Akshith-Patil — APPROVED** [review]
Nice job Peter👍

File: components/SummaryContainer.tsx (hunk 1 above)
Comment (reteppeter) [reply components/SummaryContainer.tsx:43]: Yeah, I've just made it resemble the figma. There's supposed to be an image that pushed it to the right, but I couldn't find the mui icon.

Comment (reteppeter) [reply components/SummaryContainer.tsx:46]: I've made it smaller now, and it lines up with the "Dependent Repos" box (but not the "Repos Overview" box, which is lower).

**Pull request merged**